openapi/src/common/parameters.yaml

```diff
@@ -101,14 +101,25 @@ components:
       schema:
         type: string
 
+    SasExpirationDuration:
+      name: sasExpirationDuration
+      in: query
+      description: |
+        The number of seconds until the SAS token should expire (optional). Default and maximum allowed values are
+        set in configuration.
+      required: false
+      schema:
+        type: integer
+        format: int64
```

Comment:
int64 for long

Reply:
I was conforming to the fact that the duration set in configuration is a Long (even though it is in minutes). I figured if we were concerned about that value potentially not fitting in a 32-bit int than we should certainly be concerned about seconds.

Reply:
@marctalbott do you have any insight into the choice of a Long for configuration values? If I recall correctly you wrote that code.

Reply:
Hmm, can't say I remember unfortunately. The only thing I can think of is that the `OffsetDateTime` methods seem to expect longs over ints, but that doesn't actually restrict us from using ints here